src/main/app/controller/PostController.test.ts

```diff
@@ -108,33 +145,35 @@ describe('PostController', () => {
   test('Should save the users data and redirect to the next page if the form is valid with parsed body', async () => {
     getNextStepUrlMock.mockReturnValue('/next-step-url');
     const errors = [] as never[];
-    const body = { day: '1', month: '1', year: '2000' };
+    const body = { day: '11', month: '12', year: '2000' };
     const parsedBody = {
-      date: { day: '1', month: '1', year: '2000' },
+      date: { day: '11', month: '12', year: '2000' },
```

Comment:
this seems like a bodge to avoid the 01 vs 1 issue

Reply:
👍 I've refactored this so that the `fromApiFormat` will drop any padded zeros in 559b32b.